vmdb/app/models/container.rb

```diff
@@ -4,5 +4,6 @@ class Container < ActiveRecord::Base
 
   belongs_to :container_group
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_node, :to => :container_group
```

Comment:
@simon3z I changed it to has_one :through (I think belongs_to cannot have :through)